scripts/st2bootstrap-deb.sh

```diff
@@ -329,6 +329,10 @@ install_st2() {
   sudo crudini --set /etc/st2/st2.conf database password "${ST2_MONGODB_PASSWORD}"
 
   sudo st2ctl start
+  # TODO: Fix https://github.com/StackStorm/st2-packages/issues/445
```

Comment:
Added a few more words of explanation.
But I think that dedicated issue assigned to that bug is enough.